autopart-hibernation.ks.in

```diff
@@ -0,0 +1,70 @@
+#test name: autopart-hibernation
+%ksappend repos/default.ks
+network --bootproto=dhcp
+
+%ksappend common/common_no_storage_and_payload.ks
+zerombr
+clearpart --all --initlabel
+# Include the generated autopart command.
+%include /tmp/ksinclude
+
+%packages
+%end
+
+%pre
+# Generate the autopart command.
+autopart="autopart"
+echo "$autopart" >/tmp/ksinclude
+%end
```

Comment:
I think something is missing here. As written now, this is equivalent to:
```
autopart

%packages
%end
```

The whole dance with generated include does nothing. I guess it is supposed to do something with the variable before echoing?

Reply:
Thanks, I removed the parameter to test if it fails properly and forgot to put it back in :skull: